bzkanban.js

```diff
@@ -333,6 +339,7 @@ function loadBoard() {
     clearAssigneesList();
     clearCards();
     loadBugs();
+    loadBacklogCards();
```

Comment:
Doing this here is probably too soon. If it's needed to refresh page after drag and drop, could wrap it in a check that the backlog is shown.